client/components/header-cake/index.jsx

```diff
@@ -40,16 +39,16 @@ module.exports = React.createClass( {
 		return (
 			<Card className={ classes }>
 				<div className="header-cake__corner">
```

Comment:
What's the purpose of this `<div />`? Do we think we might add to it in the future? If not, should we collapse it into `<HeaderCakeBack />` and leave it out here?